packages/api/config/lambdas.yml

```diff
@@ -89,6 +89,27 @@ kinesisConsumer:
       function: Ref
       value: ProvidersTableDynamoDB
     bucket: '{{system_bucket}}'
+    FallbackTopicArn:
+      function: Fn::Sub
+      value: '${kinesisFallbackSns}'
+
+fallbackConsumer:
```

Comment:
In order to break circular dependencies in the CF template, I had to add a separate lambda that processed the fallback SNS notifications.  Is this OK?  And the right thing to do?

Reply:
Do we need this lambda? Is it sufficient to send the messages to the SNS topic and have application owners decide how to consume them?

Reply:
I could be mistaken, but I think you need to have a subscriber on the SNS topic in order for anything to happen.  If we don't set that up, then the topic publish just happens and then no one hears it. (or that's my understanding.)